samples/01_basic_ping_bot/Program.cs

```diff
@@ -14,23 +14,31 @@ namespace _01_basic_ping_bot
     // - Here, under the 02_commands_framework sample
     // - https://github.com/foxbot/DiscordBotBase - a bare-bones bot template
     // - https://github.com/foxbot/patek - a more feature-filled bot, utilizing more aspects of the library
-    class Program
+    class Program : IDisposable
```

Comment:
Is that really necessary? On termination of the program that is already taken care of.

Reply:
This is simply not true. For example, if you use a Timer and do not dispose it, your application may not close properly if it is ran using `dotnet run`.